unittests/AST/TensorFlow.cpp

```diff
@@ -99,3 +104,78 @@ TEST_F(TypeContainsTensorFlowValueTest, WorksForRecursiveTypes) {
   recursiveDecl->addMember(varDecl);
   EXPECT_FALSE(containsTensorFlowValue(recursiveType));
 }
+
+TEST_F(TypeContainsTensorFlowValueTest, TensorFlowType) {
```

Comment:
TensorFlowType -> IsTensorHandleWorks (or something along these lines)?

Reply:
I gave more expressive name to this and other tests in the file as well. Renaming coming in my next commit.